src/AutoRest.CSharp/Common/Output/Expressions/KnownValueExpressions/BinaryDataExpression.cs

```diff
@@ -13,6 +13,8 @@ internal sealed record BinaryDataExpression(ValueExpression Untyped) : TypedValu
         public FrameworkTypeExpression ToObjectFromJson(Type responseType)
             => new(responseType, new InvokeInstanceMethodExpression(Untyped, nameof(BinaryData.ToObjectFromJson), Array.Empty<ValueExpression>(), new[] { new CSharpType(responseType) }, false));
 
+        public StringExpression ToText() => new(Invoke(nameof(BinaryData.ToString)));
```

Comment:
if we want to invoke the `ToString` method, 
1. we should name this as `ToString` and disambiguate to `InvokeToString`.
2. since every object has a `ToString` method, we should not define this method on this type. We should define it on `ValueExpression`
3. `ValueExpression` already has a method `InvokeToString`.

Therefore we should remove this method, and where this method is called, we should use `InvokeToString` instead.